fastlane/Fastfile

```diff
@@ -211,40 +217,79 @@ ENV["APP_STORE_STRINGS_FILE_NAME"]="AppStoreStrings.pot"
       ios_lint_localizations(input_dir: 'WooCommerce/Resources', allow_retry: true)
       ios_bump_version_beta()
     end
-    ios_final_tag(options)
 
     # Wrap up
     version = ios_get_app_version()
+    ios_clear_intermediate_tags(version: version)
     removebranchprotection(repository:GHHELPER_REPO, branch: "release/#{version}")
     setfrozentag(repository:GHHELPER_REPO, milestone: version, freeze: false)
     create_new_milestone(repository:GHHELPER_REPO)
     close_milestone(repository:GHHELPER_REPO, milestone: version)
+
+    # Start the build
+    trigger_release_build(branch_to_build: "release/#{version}")
   end
 
   #####################################################################################
   # build_and_upload_release
   # -----------------------------------------------------------------------------------
-  # This lane builds the app and upload it for distribution 
+  # This lane builds the app and uploads it for distribution 
   # -----------------------------------------------------------------------------------
   # Usage:
-  # bundle exec fastlane build_and_upload_release [skip_confirm:<skip confirm>] [create_gh_release:<create release on GH>]
+  # bundle exec fastlane build_and_upload_release [skip_confirm:<skip confirm>]
+  #  [create_gh_release:<create release on GH>]
   #
   # Example:
   # bundle exec fastlane build_and_upload_release 
   # bundle exec fastlane build_and_upload_release skip_confirm:true 
   #####################################################################################
-  desc "Builds and updates for distribution"
+  desc "Builds and uploads for distribution"
   lane :build_and_upload_release do | options |
-    final_tag = (is_ci() == true) ? ios_validate_ci_build() : false
-    create_release = (final_tag && is_ci()) || options[:create_gh_release]
-
     ios_build_prechecks(skip_confirm: options[:skip_confirm], 
       internal: false,
       external: true)
     ios_build_preflight()
     build_and_upload_itc(skip_prechecks: true, skip_confirm: options[:skip_confirm], create_release: create_release)
```

Comment:
`create_release` variable is now undefined at that point.
Should we make it an `option[:create_release]` and pass it at call site? or decide to always create a GH release from now on?

Reply:
Yeah, good catch.